src/marqo/tensor_search/on_start_script.py

```diff
@@ -110,11 +110,11 @@ def run(self):
 
         try:
             nltk.data.find('tokenizers/punkt')
-            self.logger.info("nltk data: tokenizers/punkt already downloaded")
+            self.logger.debug("nltk data: tokenizers/punkt already downloaded")
         except LookupError:
```

Comment:
Let's make it throw a warning here, as this indicates something is not quite right

Reply:
this is actually the expected behavior. we want it to say "tokenizers/punkt already downloaded" because that means it downloaded in build time.